keyboards/bpiphany/pegasushoof/2015/config.h

```diff
@@ -17,6 +17,13 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
 #pragma once
 
+/* USB Device descriptor parameter */
+#define VENDOR_ID       0x4245  // "BE" (Bathroom Epiphanies)
+#define PRODUCT_ID      0x6050  // "`p"
+#define DEVICE_VER      1.0
+#define MANUFACTURER    "Filco"
+#define PRODUCT         "Majestouch TKL The Pegasus Hoof 2015"
+
```

Comment:
This should not be added back, any required changes must instead be made to `info.json`
```suggestion
```